scss/variables/breakpoints.js

```diff
@@ -0,0 +1,32 @@
+( function( root, factory ) {
+	if ( typeof define === "function" && define.amd ) {
+		define( [ "./chassis" ], factory );
+	} else if ( typeof exports === "object" ) {
+		module.exports = factory( require( "./chassis" ) );
+	} else {
+		root.chassis = factory( root.chassis );
+	}
+}( this, function( chassis ) {
+
+chassis.breakpoints = {
+
+	"viewport-xs": {
```

Comment:
breakpoints make sense to me. If that will minimize confusion, I see no reason why we can't use that instead of 'viewports'